.gitignore

```diff
@@ -35,3 +35,5 @@ charts/configs/bitcoind/*
 coverage
 lib
 artifacts
+
+.node-version
```

Comment:
I hope this is ok. I use the `nodenv` package locally to manage my node virtual environments and this is an artefact from that.

Reply:
I don't think we want to allow this. I remember @bodymindarts saying stuff like this should be part of the [global](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) gitignore.

Reply:
Yup, it's since been removed